framework/src/Volo.Abp.AspNetCore.Mvc.UI.MultiTenancy/Pages/Abp/MultiTenancy/TenantSwitchModal.cshtml.cs

```diff
@@ -48,7 +48,7 @@ public async Task OnPostAsync()
             else
             {
                 var tenant = await TenantStore.FindAsync(Input.Name);
-                if (tenant == null)
+                if (tenant == null || !tenant.IsActive)
```

Comment:
I think a different exception message for the `!tenant.IsActive` would be better.